lib/improver/metadata/constants/mo_attributes.py

```diff
@@ -33,6 +33,9 @@
 GRID_TYPE = 'standard'
 STAGE_VERSION = '1.3.0'
 
+MOSG_GRID_ATTRIBUTES = {
+    'mosg__grid_type', 'mosg__grid_version', 'mosg__grid_domain'}
+
```

Comment:
Ideally this whole file would be stores as a json (or series of json) files.

Reply:
In the source code?  How is that essentially different from what's done here?

Reply:
It's not in the repository. It's like the site-init file which we use to configure site specific information. It lives in /etc but is not in the git repo.